ui/src/clockface/components/dropdowns/Dropdown.tsx

```diff
@@ -174,55 +173,52 @@ class Dropdown extends Component<Props, State> {
       children,
       testID,
     } = this.props
+
     const {expanded} = this.state
 
-    if (expanded) {
```

Comment:
Hey @alexpaxton @mavarius, could I get your opinion on this change? I removed the validation / dev assertion that a `Dropdown` must have at least 1 child.

I needed to display a dropdown that loads its items asynchronously. So it may no have any items (either hasn't been loaded, or has been loaded but no results were returned).

<img width="327" alt="Screen Shot 2019-03-18 at 8 58 59 AM" src="https://user-images.githubusercontent.com/638955/54544766-842f7680-495d-11e9-96f0-6addd161b033.png">

<img width="380" alt="Screen Shot 2019-03-18 at 8 59 08 AM" src="https://user-images.githubusercontent.com/638955/54544781-8a255780-495d-11e9-98dd-9f1b06e70628.png">